Entities/AHuman.h

```diff
@@ -925,7 +933,7 @@ ClassInfoGetters;
 //                  reticule should be drawn, to indicate force in the throw.
 // Return value:    None.
 
-	void DrawThrowingReticule(BITMAP *pTargetBitmap, const Vector &targetPos = Vector(), double amount = 1.0) const;
+	void DrawThrowingReticule(BITMAP *pTargetBitmap, const Vector &targetPos = Vector(), float progressScalar = 1.0F) const;
```

Comment:
![image](https://user-images.githubusercontent.com/50018358/132344337-6d5ffc69-0bef-4175-a91a-384a140df945.png)
lel

![image](https://user-images.githubusercontent.com/50018358/132344579-24056ddb-0cd2-4997-8a35-c57a9315c0b7.png)
Variant my ass.
Rename pls.

Reply:
fucken data man

Reply:
¯\\\_( ͡° ͜ʖ ͡°)_/¯

Reply:
this is reticulous